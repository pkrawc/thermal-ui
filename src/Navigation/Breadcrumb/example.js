import React from 'react'
import { Container } from '../../Layout'
import { Breadcrumb } from '../Breadcrumb'

const BreadcrumbExample = props =>
  <Container>
    <Breadcrumb />
  </Container>

export default {
  title: 'Breadcrumb',
  example: BreadcrumbExample
}
