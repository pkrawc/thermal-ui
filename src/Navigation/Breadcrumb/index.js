import React, { Children } from 'react'
import styled from 'styled-components'

const BreadcrumbWrapper = styled.ul`
  display: flex;
  margin: 0;
  padding: 0;
`

export const Breadcrumb = props =>
  <BreadcrumbWrapper {...props}>
    <li className="crumb">
      hi
    </li>
  </BreadcrumbWrapper>
