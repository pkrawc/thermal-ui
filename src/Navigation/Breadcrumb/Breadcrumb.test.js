import React from 'react'
import renderer from 'react-test-renderer'
import { Breadcrumb } from '../Breadcrumb'
import { BrowserRouter } from 'react-router-dom'

describe('Breadcrumb', () => {

  it('renders the Breadcrumb without crashing', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Breadcrumb />
      </BrowserRouter>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('navigates to the correct crumb when clicked', () => {
    return Promise.resolve(true)
  })
})
