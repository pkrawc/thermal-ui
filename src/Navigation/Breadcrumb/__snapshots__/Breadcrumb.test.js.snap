exports[`Breadcrumb renders the Breadcrumb without crashing 1`] = `
<ul
  className="hzgMIJ">
  <span>
    /
    blank
  </span>
</ul>
`;
